.github/workflows/deploy.yml

```diff
@@ -0,0 +1,19 @@
+name: Deploy Application
+on: [workflow_dispatch]
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Set variables
+              run: |
```

Comment:
this is basically saying: 'if workflow is triggered from master branch, then set the variables for deploying to prod; if triggered from any other branch, then set the variables for deploying to staging'.